src/java.base/share/classes/java/util/Random.java

```diff
@@ -120,7 +256,7 @@ private static long seedUniquifier() {
     }
 
     private static final AtomicLong seedUniquifier
-            = new AtomicLong(8682522807148012L);
+    = new AtomicLong(8682522807148012L);
```

Comment:
This seems (to me) like an odd indentation change.

Reply:
Reverted the wrong indentation changes sorry about that.